ingesters/ingesters.md

```diff
@@ -410,6 +410,46 @@ Common configuration errors for the Federator include:
 * Enforcing certification validation when upstream indexers or Federators do not have certificates signed by a trusted certificate authority (see the `Insecure-Skip-TLS-Verify` option)
 * Mismatched Ingest-Secret for downstream ingesters
 
+## Kafka Federator
+
+Gravwell also provides a Kafka Federator, that behaves exactly like the Federator, except that it uses Kafka as an upstream transport. Ingesters can connect to it just with Federator, and ingested entries will be put into Kafka Topics as messages. 
```

Comment:
`Ingesters can connect to it just with Federator, and ingested entries` this right?